src/main/resources/swagger.api/schemas/exportConfig.json

```diff
@@ -7,6 +7,10 @@
       "id": {
         "type": "string"
       },
+      "description": {
```

Comment:
it's not required in configuration, description is job specific

as we discussed before with Phil it should be smth like this for Bursar exports:
```
# of charges: 20
# of refunds: 50
```(as I remember)